### PR TITLE
docs: update examples doc

### PR DESCRIPTION
**Pull request by aaron276h**
## Description
Removed outdated code from examples, and added a couple FAQ items that have come up recently in the community slack. 

## Test Plan
Compiled locally.

### CODE REVIEW

File: docs/faq.txt [hunk 1]
```diff
@@ -132,4 +132,17 @@ Determined has support for TensorFlow models that use the :ref:`tf.keras <tf-ker
 or :ref:`Estimator <estimator-trial>` APIs. For models that use the low-level TensorFlow
 Core APIs, we recommend porting your model to use :ref:`Estimator Trial <estimator-trial>`.
 `Example of converting a Tensorflow graph into an Estimator
-<https://github.com/determined-ai/determined/blob/master/examples/experimental/mnist_tp_to_estimator/model_def.py>`_.
+<https://github.com/determined-ai/determined/blob/master/examples/experimental/trial/mnist_tp_to_estimator/model_def.py>`_.
+
+
+Tensorboard Support
```
Comment (neilconway) [inline docs/faq.txt:138]: "TensorBoard", here and below.

File: docs/faq.txt [hunk 2]
```diff
@@ -132,4 +132,17 @@ Determined has support for TensorFlow models that use the :ref:`tf.keras <tf-ker
...
+-------------------
+
+Can I log Tensorboards events in Determined?
```
Comment (neilconway) [inline docs/faq.txt:141]: I'm a little confused by how this is worded: isn't all TensorBoard metadata captured as a series of events? Is this talking about logging additional custom data to TensorBoard? I wonder if a code example would be helpful -- i.e., where should the path `/tmp/tensorboard` be passed to?

File: docs/faq.txt [hunk 3]
```diff
@@ -132,4 +132,17 @@ Determined has support for TensorFlow models that use the :ref:`tf.keras <tf-ker
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Yes you can log TF Events to ``/tmp/tensorboard``. For more details see the
+:ref:`Tensorboard How-To Guide <data-in-tensorboard>`.
+
+Can I use Tensorboard with PyTorch?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Yes! For an example of this check out the `mnist-GAN example <https://github.com/determined-ai/determined/blob/master/examples/official/trial/mnist_gan_pytorch/model_def.py>`_.
```
Comment (neilconway) [inline docs/faq.txt:148]: Would you mind also updating the TensorBoard How-To guide with this information? A little more detail would be nice, as well -- i.e., is using `determined.tensorboard.metric_writers.pytorch.TorchWriter` necessary? (If so, maybe we should add that to the API reference -- but we can do that later.)

File: docs/faq.txt (hunk 2 above)
Comment (aaron276h) [reply docs/faq.txt]: I rephrased this and added a code snipped showing how this can be done for PyTorch.

File: docs/faq.txt (hunk 3 above)
Comment (aaron276h) [reply docs/faq.txt]: Added a bit more detail here, also added info about PyTorch to the How-To guide and a reference guide for `determined.tensorboard.metric_writers.pytorch.TorchWriter`.

**Pull request merged**